capstone/src/main/java/com/step902020/capstone/IndividualController.java

```diff
@@ -0,0 +1,114 @@
+
+package com.step902020.capstone;
+
+import java.io.IOException;
+import java.util.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.view.RedirectView;
+import com.google.gson.Gson;
+import java.io.IOException;
+
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@RestController
+public class IndividualController {
+
+  @Autowired
+  private IndividualRepository individualRepository;
+
+  @GetMapping("get-individual")
+  public List<Individual> getIndividual(@RequestParam("email") String email) {
+    return this.individualRepository.findByEmail(email);
+  }
+
+
+  @PostMapping("save-individual")
+  public RedirectView saveIndividual(
```

Comment:
In terms of naming of methods and fields, I find the use of the word "save" to be a bit ambiguous because it could mean either "create" or "update".
For this specific method, `registerIndividual` would probably be more clear.

Reply:
Yeah I agree, but I guess the ambiguity is there because this method does either an update or a create depending on whether the individual exists already.